boot/bootutil/include/bootutil/crypto/common.h

```diff
@@ -1,19 +1,20 @@
 /*
  * SPDX-License-Identifier: Apache-2.0
  *
- * Copyright (c) 2021 Arm Limited

```

Comment:
This file also has DOS line endings, which we should probably fix. Perhaps add a single commit at the front of this sequence that fixes the line endings in this file.